src/Tickets/Flexible_Tickets/Series_Passes/Base.php

```diff
@@ -603,23 +603,24 @@ public function hide_header_image_option_from_ticket_settings( string $html = nu
 	 * Filters the start date for a series to use the start date of the first event in the series.
 	 *
 	 * @since 5.8.0
+	 * @since TBD Changed the strict type of the $display_time and $date_format parameter to mixed.
 	 *
 	 * @param string  $start_date   The start date.
 	 * @param WP_Post $series       The series post object.
-	 * @param bool    $display_time Whether to display the time.
-	 * @param string  $date_format  The date format.
+	 * @param mixed   $display_time Whether to display the time.
```

Comment:
I would not change the types in the doc-block: they represent what is _ideally_ expected.